src/main/kotlin/com/terraformation/backend/tracking/api/PlantingSubzonesController.kt

```diff
@@ -61,5 +62,6 @@ data class ListPlantingSubzoneSpeciesResponsePayload(
 data class UpdatePlantingSubzoneRequestPayload(
     val finishedPlanting: Boolean,
 ) {
-  fun applyTo(row: PlantingSubzonesRow) = row.copy(finishedPlanting = finishedPlanting)
+  fun applyTo(row: PlantingSubzonesRow): PlantingSubzonesRow =
+      row.copy(finishedTime = if (finishedPlanting) Instant.EPOCH else null)
```

Comment:
Add a comment why Instant.EPOCH? I understand it's used as a non-null flag/check for updatePlantingSubzone.